book/website/community-handbook/community-calls/community-calls-steering-committee.md

```diff
@@ -0,0 +1,15 @@
+(ch-calls-steering-committee)=
+# Steering Committee Meetings 
+
+```{warning} This page is under review and will be updated with more details soon.
+```
+## Steering Committee Meetings
+In 2025, Steering Committee Meetings will be hosted regularly to formalise Constitution Level governance.
+These meetings are attended by drawing from chairs and representatives of Working Groups and Delivery Groups (projects carried out in collaboration with _The Turing Way_).
+These calls utilise prepared agenda items and particular issues to be voted on by different members.
+The Chair and Secretary of the Steering Committee are responsible for scheduling and agreeing the agenda. 
```

Comment:
I have a recollection of reading something similar. Maybe it was in a framapad of proposed governance. I just want to make sure we don't duplicate this information.

Reply:
see the comment in the other document - we should have tables/roles somewhere central but we currently don't so for now I think this will be okay?